sampleserver/sampleserver.go

```diff
@@ -58,6 +61,31 @@ type server struct {
 	pathSeq int32
 }
 
+func (ss *server) onCStoreStream(
+	transferSyntaxUID string,
+	sopClassUID string,
+	sopInstanceUID string,
+	dataCh chan []byte) dimse.Status {
+	buf := bytes.Buffer{}
+
+	e := dicomio.NewEncoderWithTransferSyntax(&buf, transferSyntaxUID)
```

Comment:
Pls write this into local file like `func (ss *server) onCStore`

Reply:
If you want to check with the old `onCStore`, please checkout `segmed:master` and build it yourself. We don't use the same logic for `onCStore` here.